flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/SubtasksTimesInfo.java

```diff
@@ -132,13 +138,14 @@ public boolean equals(Object o) {
             SubtaskTimeInfo that = (SubtaskTimeInfo) o;
             return subtask == that.subtask
                     && Objects.equals(host, that.host)
```

Comment:
will host be `(unassigned)` if not set like the deprecated location? 
I assume that as the old form did not have the port, the old form and new form cannot ever be equivalent.

Reply:
Yes, it will. The old form and the new form are equivalent in this case.